src/launchpad/size/insights/apple/alternate_icons_optimization.py

```diff
@@ -71,7 +71,6 @@ def _resize_icon_for_analysis(self, img: Image.Image) -> Image.Image:
         )
 
     def _is_alternate_icon_file(self, file_info: FileInfo, alternate_icon_names: set[str]) -> bool:
-        return (
-            file_info.file_type.lower() in self.OPTIMIZABLE_FORMATS
-            and Path(file_info.path).stem in alternate_icon_names
-        )
+        # Some asset catalog entries have no extension, so we include "other" in the OPTIMIZABLE_FORMATS
+        file_type_match = file_info.file_type.lower() in (self.OPTIMIZABLE_FORMATS | {"other"})
```

Comment:
The "other" check doesn't seem ideal, is file_type not showing as PNG for these?